testsupport/tiers/checks.go

```diff
@@ -218,8 +218,8 @@ type base1nsTierChecks struct {
 
 func (a *base1nsTierChecks) GetNamespaceObjectChecks(_ string) []namespaceObjectsCheck {
 	checks := []namespaceObjectsCheck{
-		resourceQuotaComputeDeploy("20", "14Gi", "1750m", "14Gi"),
-		resourceQuotaComputeBuild("20", "10Gi", "2", "10Gi"),
+		resourceQuotaComputeDeploy("20", "14Gi", "3000m", "14Gi"),
```

Comment:
Due to internal resource quotas object specifics we have to use full core numbers instead of the minicores in tests.
```suggestion
		resourceQuotaComputeDeploy("20", "14Gi", "3", "14Gi"),
```